### PR TITLE
Correct mismatch between ATDM_CONFIG_COMPILER=INTEL-19.0.5 and module load sems-intel/19.0.5

### PR DESCRIPTION
**Pull request by PhilMiller**

### CODE REVIEW

File: cmake/std/atdm/contributed/kokkos-dev-2/environment.sh [hunk 1]
```diff
@@ -196,7 +196,7 @@ elif [[ "$ATDM_CONFIG_COMPILER" == "INTEL-18.0.5" ]] ; then
   export ATDM_CONFIG_BLAS_LIBS="-mkl"
 elif [[ "$ATDM_CONFIG_COMPILER" == "INTEL-19.0.5" ]] ; then
   module load sems-archive-gcc/7.2.0
-  module load sems-archive-intel/18.0.5
+  module load sems-archive-intel/19.0.5
   module load atdm-env
   module load atdm-mkl/18.0.5
```
Comment (jhux2) [inline cmake/std/atdm/contributed/kokkos-dev-2/environment.sh:201]: Should the mkl version also be changed?

Comment (jhux2) [inline cmake/std/atdm/contributed/kokkos-dev-2/environment.sh:201]: Should the mkl version also be changed?

Comment (PhilMiller) [reply cmake/std/atdm/contributed/kokkos-dev-2/environment.sh:201]: No, SEMS doesn't have a corresponding mkl module for the later version.

**Review by e10harvey — COMMENTED** [review]
Thanks, @PhilMiller ! Can you please post a cdash link to the test results?

**Comment by PhilMiller** [comment]
I haven't bothered to run it.I guess being proactive on detecting and reporting any errors revealed would be good form.

**Review by ndellingwood — APPROVED** [review]
Let's get the typo fix and update in, we can add more updates for this machine in follow up pull request if needed

**Pull request merged**